modules/adding-an-application-to-the-dashboard.adoc

```diff
@@ -0,0 +1,68 @@
+:_module-type: PROCEDURE
+
+[id="adding-an-application-to-the-dashboard_{context}"]
+= Adding an application to the dashboard
+
+[role='_abstract']
+If you have installed another application in your {openshift-platform} cluster, you can add a tile for that application to the {productname-short} dashboard (the *Applications* -> *Enabled* page) to make it easily accessible for users. 
+
+.Prerequisites
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+
+.Procedure
+. Log in to the {openshift-platform} console as a cluster administrator.
+. In the {openshift-platform} console, click *+* in the top navigation bar.
+. Select your project from the list of projects.
+. Copy the following example code and paste it into the *Import YAML* editor.
++
+[.lines_space]
+[.console-input]
+[source, yaml]
+----
+apiVersion: dashboard.opendatahub.io/v1
+kind: OdhApplication
+metadata:
+  annotations:
+    opendatahub.io/categories: 'Data management,Data preprocessing'
+  name: examplename
+  namespace: redhat-ods-applications
+  labels:
+    app: odh-dashboard
+    app.kubernetes.io/part-of: odh-dashboard
+spec:
+  enable:
+    validationConfigMap: examplename-enable
+  img: >-
+    <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="path data" fill="#ee0000"/>
+    </svg>
+  getStartedLink: 'https://example.org/docs/quickstart.html'
+  route: exampleroutename
+  routeNamespace: examplenamespace
+  displayName: Example Name
+  kfdefApplications: []
+  support: third party support
+  csvName: ''
+  provider: example
+  docsLink: 'https://example.org/docs/index.html'
+  quickStart: ''
+  getStartedMarkDown: >-
+    # Example
+
+    Enter text for the information panel.
+
+  description: >-
+    Enter summary text for the tile.
+  category: Self-managed
+----
+
+. Modify all the parameters in the example code for your application.
```

Comment:
is it really ALL the parameters? I think some parameters will always have the same values, for example:
apiVersion: dashboard.opendatahub.io/v1
kind: OdhApplication